test/404s.test.js

```diff
@@ -1772,3 +1772,26 @@ test('setNotFoundHandler should be chaining fastify instance', t => {
 
   t.end()
 })
+
+test('Should handle to invoke callNotFound inside a 404 handler', t => {
+  t.plan(2)
+
+  let fastify = null
+  try {
+    fastify = Fastify({
+      frameworkErrors: (error, req, reply) => {
+        return reply.callNotFound(error)
+      }
+    })
+  } catch (e) {
+    t.fail()
+  }
```

Comment:
Please remove the try/catch, it's not needed for this test

Reply:
Done